### DISCUSSION TITLE
can i request a few title covers for Aurora?

### DISCUSSION THREAD

**Discussion by maurinahuel62** [XboxUnity/XboxUnity#2] [General]
i got state of Emergency and WWF Raw on my aurora 
and i Can't figure out a way to add game covers to them.
my username is maurinahuel on xboxunity.

sorry for the troubles, cheers.

**Comment by Swizzy** [comment]
If you have covers for the games, feel free to upload them and they will be reviewed by staff.

You can also manage your covers using this tool; https://github.com/XboxUnity/AuroraAssetEditor
